components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/context/IdentityContext.java

```diff
@@ -56,6 +56,7 @@ public static IdentityContext getThreadLocalIdentityContext() {
      *
      * @param flow flow of the request.
      */
+    @Deprecated // Use enterFlow() and exitFlow() methods for flow management.
```

Comment:
Let's add the comment to doc comment.
E.g.,:
`@deprecated Use {@link #enterFlow()} instead.`

Reply:
Fixed f8765c8803d0aa97aed9587f78243a3818978b54